src/mscorlib/model.xml

```diff
@@ -1938,12 +2008,82 @@
       <Member MemberType="Property" Name="TickCount" />
       <Member MemberType="Property" Name="Version" />
       <Member MemberType="Property" Name="ExitCode" />
+      <Member MemberType="Property" Name="CurrentDirectory" />
+      <Member MemberType="Property" Name="Is64BitOperatingSystem" />
+      <Member MemberType="Property" Name="Is64BitProcess" />
+      <Member MemberType="Property" Name="SystemPageSize" />
+      <Member MemberType="Property" Name="UserDomainName" />
+      <Member MemberType="Property" Name="UserInteractive" />
+      <Member MemberType="Property" Name="UserName" />
+      <Member MemberType="Property" Name="WorkingSet" />
       <Member Status="ImplRoot" Name="GetResourceStringLocal(System.String)" />
       <Member Status="ImplRoot" Name="SetCommandLineArgs(System.String[])" />
       <Member Name="FailFast(System.String)"  />
       <Member Name="FailFast(System.String,System.Exception)"  />
       <Member Name="Exit(System.Int32)" />
     </Type>
+    <Type Name="System.Environment+SpecialFolder">
```

Comment:
Similarly, we define this in corefx; we shouldn't need to expose it from CoreCLR.